$NetBSD: patch-ae,v 1.1 2007/03/17 16:50:50 rillig Exp $

gcc4 complains:
main.c:50: error: static declaration of 'prog_name' follows non-static declaration
sawmill_subrs.h:54: error: previous declaration of 'prog_name' was here

--- src/main.c.orig	2002-04-10 05:59:54.000000000 +0200
+++ src/main.c	2007-03-17 17:46:34.000000000 +0100
@@ -46,9 +46,6 @@
 /* one of the ec_ values */
 int exit_code = ec_no_exit;
 
-/* Saved value of argv[0] */
-static char *prog_name;
-
 DEFSYM(sawfish_directory, "sawfish-directory");
 DEFSYM(sawfish_lisp_lib_directory, "sawfish-lisp-lib-directory");
 DEFSYM(sawfish_site_lisp_directory, "sawfish-site-lisp-directory");
@@ -333,6 +330,7 @@ main(int argc, char **argv)
     char **old_argv;
     int old_argc;
     char *lang;
+    const char *prog_name;
 
     /* This will fork multiple copies, if necessary */
     multihead_init (&argc, &argv);
